src/PresentationalComponents/CreateImageWizard/WizardStepUploadGoogle.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form, FormGroup, TextList, TextListItem, Popover, Radio, TextContent, Text, TextInput, Title } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+const WizardStepUploadGoogle = (props) => {
+    const accountTypePopover = (
+        <Popover
+            hasAutoWidth
+            maxWidth='35rem'
+            headerContent={ 'Valid types of account emails' }
```

Comment:
Although 3 out of 4 types are really emails, maybe we could reword this to "Valid account types"?

Reply:
Good point! I will change this.